### PR TITLE
Spi update

### PR DESCRIPTION
**Pull request by Sellto**
no comment.

### CODE REVIEW

File: src/electronics/communication/communication.md [hunk 1]
```diff
@@ -0,0 +1 @@
+##This section aims to show how different communication techniques have been implemented
```
Comment (azerupi) [inline src/electronics/communication/communication.md]: You forgot the spaces between the hashtags and your titles 😉

File: src/electronics/communication/i2c.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#I2C
```
Comment (bluebeel) [inline src/electronics/communication/i2c.md]: Espace :)

File: src/electronics/communication/spi.md [hunk 3]
```diff
@@ -190,11 +190,43 @@ switch (mySPI.command) {
...
+
+Patch : The data is sent by ROS every 50ms even if it is not changed (see on [motor_sim.py](https://github.com/Ecam-Eurobot/Eurobot-2018/blob/master/ros_packages/mecanum/src/motor_sim.py))
+
+
 ##### 3. Changing the values of a register when reading another register
+When we want to read the return of a slave encoder we send a series of null bytes after the address byte to receive the value. But these null values are misinterpreted and the engine speed drops to zero. Probably from a coding error.
+
+Patch : We send velocity value instead of null bytes. (just using during the Eurobot sprint)
+
+#Just for curious
```
Comment (bluebeel) [inline src/electronics/communication/spi.md]: Espace

**Review by bluebeel — CHANGES_REQUESTED** [review]
Espace

**Review by azerupi — APPROVED** [review]

**Comment by Sellto** [comment]
here you are :)

File: src/electronics/communication/spi.md (hunk 3 above)
Comment (bluebeel) [inline src/electronics/communication/spi.md]: Espace?

**Review by bluebeel — CHANGES_REQUESTED** [review]
Espace line 219

**Review by bluebeel — APPROVED** [review]

**Pull request merged**